content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Knowledge in PostgreSQL, PHP, and Object-Oriented Programming in PHP 
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## PHP & PostgreSQL
+
+PostgreSQL is an open source object-relational database system and PHP is a scripting language designed for web development. This course will cover the basics of PostgreSQL programming with PHP Data Objects(PDO). It requires `PG PHP SQL extension` for PHP and PostgreSQL to communicate.
```

Comment:
Please rephrase the first sentence. I think it is better to talk about how DBMS in PHP work than simply defining PostgreSQL and PHP which is already stated in our previous courses. Please check. Thanks.